package.json

```diff
@@ -14,22 +13,25 @@
     "karma-coverage": "^1.1.2",
     "karma-edge-launcher": "^0.4.2",
     "karma-firefox-launcher": "^1.1.0",
-    "karma-ie-launcher": "^1.0.0",
     "karma-jasmine": "^1.1.2",
     "karma-jasmine-html-reporter": "^1.3.1",
     "karma-jasmine-spec-tags": "^1.0.1",
     "karma-verbose-reporter": "^0.0.6",
     "puppeteer": "^1.7.0",
     "requirejs": "^2.3.6",
+    "rollup": "^0.65.2",
     "shx": "^0.3.2",
-    "whatwg-fetch": "^2.0.4",
+    "terser": "^3.8.2",
     "xhr2": "^0.1.4"
   },
   "scripts": {
+    "rollup": "cd make-it && rollup",
+    "terser": "cd make-it && terser",
     "test": "karma start --browsers ChromeHeadlessNoSandbox,FirefoxHeadless",
+    "test-min": "karma start karma.min.conf.js --browsers ChromeHeadlessNoSandbox,FirefoxHeadless",
+    "test-debug": "karma start karma.debug.conf.js --browsers ChromeHeadlessNoSandbox,FirefoxHeadless",
     "test-dev": "karma start --browsers ChromeHeadlessNoSandbox --skip-tags Slow",
```

Comment:
Should this have FirefoxHeadless like the others?

Reply:
nah, `npm run test-dev` is intended for local development time as a quick way to run a subset of tests. It intentionally skips slow tests and runs only in one browser.